### DISCUSSION TITLE
Tid (copying tids) not working, and other questions

### DISCUSSION THREAD

**Discussion by Marxsal** [buggyj/tiddlyclip-plugin#57] [Q&A] [answered]
When I try to clip tids, it always tells me "not a tiddler". It tells me this whether I try to choose by selection, name, or tag.

What are the four color bands in the drop down?

Linux, Chromium 99, plugin 1.17.17, ext 0.1.8

**Comment by buggyj** [comment, marked as answer]
TW version 5.2.0 moved the tiddler store from html (web page text) into JSON (web page code). Unfortunately this means that the addon can no longer read tiddlers directly. TW versions 5.1.23 or less are not affected.

However it is possible to query a TW once it has been docked, so for TW versions 5.2.0 and greater, you need to dock to a TW before using the TID action (copying tiddlers).

I have put the answer to your other querry here: https://github.com/buggyj/tiddlyclip-plugin/discussions/59
